code/src/components/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import './Footer.css'
+
+export const Footer = () => {
+  return (
+    <footer>
+      <p className='footer-info'> This is a project made via the Technigo front-end bootcamp, Fall 2020. 
+      <br></br>My <a href='https://github.com/idsintehittapa'>GitHub</a>
```

Comment:
We shouldn't use `br` tags at all, needs to be fixed

Reply:
Fixed